python/ray/serve/tests/conftest.py

```diff
@@ -14,7 +14,7 @@
 
 
 @pytest.fixture(scope="session")
-def _shared_serve_instance():
+def shared_serve_instance():
```

Comment:
what's the context for this change?

Reply:
ditto, i found it that we need to import both `serve_instance` and `shared_serve_instance` in order to let test pass

Reply:
That shouldn't be the case, please revert this so we are making sure the test is setup properly.